__tests__/app.test.js

```diff
@@ -123,6 +149,35 @@ describe("/api/articles/:article_id", () => {
         expect(message).toBe("id not found");
       });
   });
+
+  test("PATCH 400: Respond with an error when passed an article_id with an incorrect format", () => {
+    const newArticleVote = {
+      inc_votes: 20,
+    };
+    return request(app)
+      .patch("/api/articles/invalid_id_format")
+      .send(newArticleVote)
+      .expect(400)
+      .then(({ body }) => {
+        const { message } = body;
+        expect(message).toBe("invalid id type");
+      });
+  });
+
+  test("PATCH 404: Respond with an error when passed an article_id that is not presented in the database.", () => {
+    const newArticleVote = {
+      inc_votes: 20,
+    };
+
+    return request(app)
+      .patch("/api/articles/9999")
+      .send()
+      .expect(404)
+      .then(({ body }) => {
+        const { message } = body;
+        expect(message).toBe("id not found");
+      });
+  });
 });
```

Comment:
what if something is wrong with the object you are trying to send? I.e. inc_votes has a value of 'twenty'? There's also the thought that something could be added to that object - would that be considered an error?